sql/hive/src/test/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -669,7 +669,7 @@ private[sql] class TestHiveSessionStateBuilder(
 }
 
 private[hive] object HiveTestJars {
-  private val repository = SQLConf.ADDITIONAL_REMOTE_REPOSITORIES.defaultValueString.split(",")(0)
+  private val repository = SQLConf.ADDITIONAL_REMOTE_REPOSITORIES.defaultValueString.split(",").last
```

Comment:
Out of curiosity why is this needed - the staging repo?

Reply:
Need download jars from `https://repository.apache.org/content/repositories/staging/`.

Reply:
OK so when it's released we can remove this? seems fine yes.